### DISCUSSION TITLE
Heading problem

### DISCUSSION THREAD

**Discussion by raknal** [flybywiresim/aircraft#2751] [Help] [answered]
Using the latest stable version. FMC Flightplan takeoff  enable AP climb out works using altitude constraint .
speed constraint ok, but the aicraft will not follow flightplan heading, it just keeps turning 360 degrees constantly, try to fix by 
disconnect AP and using a manually enter heading without success.
Any assistance appreciated
Richard

**Comment by ImenesFBW** [comment, marked as answer]
You need to set your deadzones. It sounds crazy but the values of your input device leak through and mess with the AP. It will depend on your device, but try setting your deadzones to 20%, and see if that fixes your problem. After you confirm that is the problem lower the DZ until it' stops working and keep it above that. This is a known issue and is the same on default.

**Comment by raknal** [comment]
Thanks for your reply

I use a basic logitech stick and don't have this issue with other stock 
aircraft nor did I have the problem prior to the latest update with your 
mod.
So is the issue only with the curent A320 mod
Deadzones are currently 2% 0% 2% 2%
Do I increase them all to 20% then decrease each deadzone down or just 
specific deadzones. The extremity deadzones are all 0%

Thank you for your assistance

Richard

*You need to set your deadzones. It sounds crazy but the values of your 
input device leak through and mess with the AP. It will depend on your 
device, but try setting your deadzones to 20%, and see if that fixes 
your problem. After you confirm that is the problem lower the DZ until 
it' stops working and keep it above that. This is a known issue and is 
the same on default.*

*—*

**Reply by ImenesFBW** [reply]
Interesting. No the issue I speak of also happens on the default A320.
For most people, 2% is enough some might need as high as 20%. 
I would set it high first just to check if this is the bug you are experiencing then go down closer to two. 
If you are turning in a circle and not also going down, the aileron axis will be the most important one to set.

Good luck! Also, feel free to visit our discord for more help at https://discord.com/invite/flybywire in the help channel

**Reply by raknal** [reply]
Changed the deadzones to 18 and all is good. Just a slight pull to the right when taxiing.
Just completed a night flight LROP - LLBG , absolutely flawless , best flight since putchasing the sim in  August
Thank you to all involved at FBW

Richard
